.github/workflows/publish-service-artifacts.yml

```
@@ -0,0 +1,22 @@
+name: Publish Service Artifacts
+
+on:
+  workflow_dispatch: 
+    inputs:
+      subpackages:
+        description: 'Subpackages to be built individually (e.g. monolith config ec2)'
+        default: 'monolith'
+        required: false
+      version:
+        description: 'Provider version (e.g. v0.1.0)'
+        required: true
+
+jobs:
+  publish-service-artifacts:
+    uses: upbound/uptest/.github/workflows/provider-publish-service-artifacts.yml@main
+    with:
+      subpackages: ${{ github.event.inputs.subpackages }}
+      version: ${{ github.event.inputs.version }}
+    secrets:
+      UPBOUND_MARKETPLACE_PUSH_ROBOT_USR: ${{ secrets.UPBOUND_MARKETPLACE_PUSH_ROBOT_USR }}
+      UPBOUND_MARKETPLACE_PUSH_ROBOT_PSW: ${{ secrets.UPBOUND_MARKETPLACE_PUSH_ROBOT_PSW }}
```

We will need to use separate robot accounts (and hence different credentials) for publishing to the release candidate repositories:
```suggestion
      UPBOUND_MARKETPLACE_PUSH_ROBOT_USR: ${{ secrets.UPBOUND_MARKETPLACE_PUSH_ROBOT_USR_RC }}
      UPBOUND_MARKETPLACE_PUSH_ROBOT_PSW: ${{ secrets.UPBOUND_MARKETPLACE_PUSH_ROBOT_PSW_RC }}
```